cmd/port-layer-server/main.go

```diff
@@ -72,7 +73,31 @@ func main() {
 		os.Exit(1)
 	}
 
-	pprof.StartPprof("portlayer server", pprof.PortlayerPort)
+	logcfg := viclog.NewLoggingConfig()
+	if ploptions.PortLayerOptions.Debug {
+		logcfg.Level = log.DebugLevel
+	}
+
+	if ploptions.PortLayerOptions.SyslogAddr != nil {
+		u, err := url.Parse(*ploptions.PortLayerOptions.SyslogAddr)
```

Comment:
is it better to use URL format for SyslogAddr? then you don't need to parse it here